apps/prairielearn/elements/pl-file-upload/pl-file-upload.js

```diff
@@ -331,7 +331,7 @@
 
     addWarningMessage(message) {
       var $alert = $(
-        '<div class="alert alert-warning alert-dismissible" role="alert"><button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>',
+        '<div class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button></div>',
```

Comment:
You lost a whole div in this conversion...

Reply:
Addressed in a4f60e9.